cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -580,6 +580,7 @@ func (g *genConversion) generateConversion(inType, outType *types.Type, sw *gene
 		}
 	} else {
 		// Emit a public conversion function.
+		sw.Do("// "+nameTmpl+" is an autogenerated conversion function. \n", args)
```

Comment:
And sounds great, I will revert those version related changes and as you said that should make Jenkins happy. Thanks!!